props/jetpack_Oli_buttons.h

```diff
@@ -0,0 +1,945 @@
+#if NUM_BUTTONS == 1
+      case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_ON):
+      // Color Change mode (2 buttons):
+#elif NUM_BUTTONS == 2
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_CLICK_SHORT, MODE_ON):
+#elif // Color Change mode (3 buttons):
+      case EVENTID(BUTTON_AUX2, EVENT_FIRST_CLICK_SHORT, MODE_ON):
+#endif
+        ToggleColorChangeMode();
+        return true;
+#endif // DISABLE_COLOR_CHANGE
+
+#if NUM_BUTTONS == 1
+      // Start or Stop Track (1 button):
+      case EVENTID(BUTTON_POWER, EVENT_FIRST_HELD_LONG, MODE_ON):
+#ifdef JETPACK_TRACK_PLAYER
+        if (!istrack_playing_)
+          StartTrackPlayer();
+        else
+          StopTrackPlayer();
+#else
+        StartOrStopTrack();
+#endif
+        return true;
+#endif // NUM_BUTTONS == 1
+
+#ifdef JETPACK_TRACK_PLAYER
+      // Play Next Track (2 & 3 buttons):
+#if NUM_BUTTONS > 1
+      case EVENTID(BUTTON_AUX, EVENT_SECOND_HELD_LONG, MODE_OFF):
+#else
+      // Play Next Track (1 button):
+      case EVENTID(BUTTON_POWER, EVENT_FOURTH_HELD_LONG, MODE_ON):
+#endif
+        NextTrack();
+        return true;
+
+      // Play Previous Track (2 and 3 buttons):
+#if NUM_BUTTONS > 1
+      case EVENTID(BUTTON_AUX, EVENT_SECOND_SAVED_CLICK_SHORT, MODE_OFF):
+#else
+      // Play Previous Track (1 button):
+      case EVENTID(BUTTON_POWER, EVENT_FOURTH_SAVED_CLICK_SHORT, MODE_ON):
+#endif
+        PreviousTrack();
+        return true;
+#endif // JETPACK_TRACK_PLAYER
+
+#if NUM_BUTTONS == 1
+      // Enter/Exit Volume MENU (1 button):
+      case EVENTID(BUTTON_NONE, EVENT_SECOND_HELD, MODE_ON):
+#else
+      // Enter/Exit Volume MENU (2 and 3 buttons):
+      case EVENTID(BUTTON_AUX, EVENT_SECOND_HELD, MODE_ON):
+      case EVENTID(BUTTON_AUX, EVENT_SECOND_HELD, MODE_OFF):
+#endif // NUM_BUTTONS == 1
+        if (!mode_volume_) {
+          mode_volume_ = true;
+          if (!hybrid_font.PlayPolyphonic(&SFX_vmbegin)); beeper.Beep(0.5, 3000);
+          PVLOG_NORMAL << "Enter Volume Menu\n";
+        } else {
+          mode_volume_ = false;
+          if (!hybrid_font.PlayPolyphonic(&SFX_vmend)); beeper.Beep(0.5, 3000);
+          PVLOG_NORMAL << "Exit Volume Menu\n";
+        }
+        return true;
+
+      // Battery level:
+#if NUM_BUTTONS == 1
+      // 1 button
+      case EVENTID(BUTTON_POWER, EVENT_THIRD_SAVED_CLICK_SHORT, MODE_OFF):
+#else
+      // 2 and 3 buttons
+      case EVENTID(BUTTON_AUX, EVENT_FIRST_HELD_LONG, MODE_OFF):
+#endif // NUM_BUTTONS == 1
+        talkie.SayDigit((int)floorf(battery_monitor.battery()));
+        talkie.Say(spPOINT);
+        talkie.SayDigit(((int)floorf(battery_monitor.battery() * 10)) % 10);
+        talkie.SayDigit(((int)floorf(battery_monitor.battery() * 100)) % 10);
+        talkie.Say(spVOLTS);
+        return true;
+    }  // switch (EVENTID)
+    return false;  // No action
+  }  // Event2
+
+  // I want the effects to play one after the other without overlapping or interrupting one another.

```

Comment:
Ok, you want something, but what does that mean?
(This comment doesn't seem very helpful.)